schema.sql

```diff
@@ -37,4 +37,30 @@ ALTER TABLE animals
 COMMIT;
 SELECT * FROM animals;
 
+/*day-4*/
+CREATE TABLE vets (
+    id INT primary key generated always as identity,
+    name VARCHAR(80) NOT NULL,
+    age INT NOT NULL,
+	date_of_graduation DATE
+);
+
+CREATE TABLE specializations (
+    id INT primary key generated always as identity,
+    vets_id INT NULL REFERENCES vets(id),
+	species_id INT NULL REFERENCES species(id)
+);
+
+CREATE TABLE visits (
+    id INT primary key generated always as identity,
+    vets_id INT NULL REFERENCES vets(id),
+	animals_id INT NULL REFERENCES animals(id),
+	visit_date DATE NULL
+);
+
+/* Block-2 */
+/*Day-1*/
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
 
+-- Create index to increase performance speed
```

Comment:
Kindly add indexes to `visits` and `owners` here in the schema file.
 ```sql
   CREATE INDEX ON visits (vet_id);
   CREATE INDEX ON owners (email);
```

Reply:
Sorry for the mistake which happened here!

I was added but during the conflict resolving was removed by accident.
Added again.